v2_1/ws/rest/docs/4_8_maintenance.md

```diff
@@ -0,0 +1,490 @@
+## Applying the HTTP methods to the SDMX REST API for Structural Metadata maintenance
+
+The following sections describe the foreseen HTTP methods for the maintenance of SDMX Structural Metadata via the REST API.
+
+### CREATE
+
+Inserting or creating new SDMX Structural Metadata is detailed in this secion.
+
+#### Artefact (Structure)
+
+Submitting SDMX Artefacts in bulk, either of the same or of different types, is achieved with an HTTP `POST` method.
+Creating new Artefact(s) may be issued by:
+- `POST` one or more Maintainable Artefacts under the proper resource type, e.g. for Codelists: `/structure/codelist/`
+- `POST` one or more Maintainable Artefacts under the abstract structure resource type, e.g. `/structure/`
+
+##### Client
+
+In order to create Artefacts, the client:
+
+- MAY set the `Accept` header to indicate the preferred response format;
+- MUST set the `Content-type` header according to the format of the submitted Artefact(s);
+- MUST include in the request body, one or more Maintainable Artefacts in the SDMX format indicated in the `Content-type` header and of the SDMX type indicated in the resource, i.e.:
+  - any set of Maintainable Artefacts under resource `/structure/`
+  - a set of specific type of Maintainable Artefacts under the corresponding resource type, e.g. for Codelists: `/structure/codelist/`
+
+##### Server
+
+In response to Artefact(s) creation, the server:
+
+- MUST return `201` upon successful creation (or `207` for partial success);
+- MUST return a `SubmitStructureResponse` message with the result of the action(s), according to the `Accept` header, or the default, if the `Accept` type is not supported (currently available only in SDMX-ML format);
```

Comment:
What is the purpose of the `Accept` header if the only supported format is SDMX-ML? Should we remove this information until the response can be returned in multiple formats?

Reply:
While you are right, I would prefer that we keep it as guidance, since any implementor could also define his own type. Maybe we could simply remove the comment in the brackets, since the equivalent messages in other formats will be produced eventually.